include/tgr_3d.xml.h

```diff
@@ -4,11 +4,11 @@
 /* Autogenerated file, DO NOT EDIT manually!
 
 This file was generated by the rules-ng-ng headergen tool in this git repository:
-http://github.com/envytools/envytools/
-git clone https://github.com/envytools/envytools.git
+http://0x04.net/cgit/index.cgi/rules-ng-ng
+git clone git://0x04.net/rules-ng-ng
```

Comment:
Seems envytools doesn't re-gen the header entirely on each update. I removed old tgr_3d.xml.h and it is fine now.